javaslang-benchmark/src/test/java/javaslang/collection/VectorBenchmark.java

```diff
@@ -43,16 +47,17 @@
 
     public static void main(String... args) {
         JmhRunner.runDebugWithAsserts(CLASSES);
-        JmhRunner.runNormalNoAsserts(CLASSES, JAVA, FUNCTIONAL_JAVA, PCOLLECTIONS, ECOLLECTIONS, CLOJURE, SCALA, JAVASLANG);
+        JmhRunner.runNormalNoAsserts(CLASSES, /*FUNCTIONAL_JAVA, PCOLLECTIONS, ECOLLECTIONS,*/ JAVA, CLOJURE, SCALA, JAVASLANG);
     }
 
     @State(Scope.Benchmark)
     public static class Base {
-        @Param({ "32", "1024", "32768" /*, "1048576", "33554432", "1073741824" */ }) /* i.e. depth 1,2,3(,4,5,6) for a branching factor of 32 */
+        @Param({ "100", "10000" })
```

Comment:
1.  I don't mind if the benchmark correctness tests (e.g. that `prependAll` concats to the start and not the end, i.e. that's I'm testing the speed of the correct thing) run only on the CI, an not locally.
2. The property based tests are incredibly useful. I have made them quicker now, they all run in 7 seconds :)
3. same as 1.

4. we could simply try parallel tests, as we're basically stateless, everything should work out of the box